server/requirements.txt

```diff
@@ -0,0 +1,2 @@
+uvicorn
+psycopg2
```

Comment:
In theory `pip install .[postgres]` should pick up this dependency (it's listed as an optional dep in the pyproject.toml), but not if I've just told you to use `--no-deps` for that part above...

Reply:
The dependencies actually want to be installed in a separate step from the pip install of the application itself -- if the requirements.txt files don't change, Docker can cache the intermediate dependency image and save a few minutes on the next build.

Also the requirements will need to be pinned in future so we won't want pyproject.toml involved.